stack.yaml

```diff
@@ -73,5 +73,6 @@ extra-deps:
 # compiler-check: newer-minor
 
 nix:
+  enable: true
```

Comment:
I presume we need this because of `Dhall`?

Reply:
Yes!